eva/executor/executor_utils.py

```diff
@@ -18,6 +18,10 @@
 from eva.models.storage.batch import Batch
 
 
+class ExecutorError(Exception):
```

Comment:
Where is the `ExecutorError` used?

Reply:
Not being used right now. I added it in the hope of using it for reporting error messages during execution.